homeassistant/components/jellyfin/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Jellyfin",
```

Comment:
Don't specify title in translation strings if it's a brand name, unless it needs to be translated.
The title defaults to the integration name in the manifest.
```suggestion
```

Reply:
Thanks! Done.